core/server/common/src/main/java/alluxio/master/journal/checkpoint/CheckpointName.java

```diff
@@ -39,4 +39,5 @@ public enum CheckpointName {
   TABLE_MASTER_TRANSFORM_MANAGER,
   TO_BE_PERSISTED_FILE_IDS,
   TTL_BUCKET_LIST,
+  CMD_ENTRY,
```

Comment:
Seems like the convention here is to not include "ENTRY" in the enum. Let's just call this "LOAD_DIRECTORY" for now.